### ISSUE TITLE
Request Support for Trakt and Simkl

### ISSUE THREAD

**Issue by Infinitay** [erengy/taiga#792]
Started watching anime recently and I don't use MAL or any other site that tailors **only** to anime to track. Instead, I use Trakt as it keeps track of not only anime, but also tv shows and movies. Also, there is another tracker called Simkl which seems to be doing the same thing as Trakt, but they offer support for plenty of applications and offer more features than Trakt.

If support for either or both of those services could be offered that would be wonderful.

**Service**: [Trakt](https://trakt.tv)
**API:** https://trakt.docs.apiary.io/

**Service**: [Simkl](https://simkl.com/)
**API:** https://simkl.docs.apiary.io/

**Comment by jsmiith** [comment]
This would be a rather cool feature.

**Comment by MrPiethon** [comment]
Agreed. The more the merrier. If a service goes down, it's good to have multiple backups.

**Comment by Gliscor** [comment]
This would be awesome! Please add support for both platforms.
